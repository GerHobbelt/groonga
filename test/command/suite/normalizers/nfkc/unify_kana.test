#@add-substitution /"version",\s"(?:.+?)"/ "\"version\", \"#{ENV['NFKC'] || '10.0.0'}\"" "\"version\", \"NFKC_VERSION\""
normalize \
  'NormalizerNFKC("unify_kana", true, \
                  "version", "NFKC_VERSION")' \
  "あイｳｪおざジたチなニぱピまミヽヾ" \
  WITH_TYPES
#@remove-substitution /"version",\s"(?:.+?)"/
