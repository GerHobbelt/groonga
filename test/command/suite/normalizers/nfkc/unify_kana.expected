normalize   'NormalizerNFKC("unify_kana", true,                   "version", "NFKC_VERSION")'   "あイｳｪおざジたチなニぱピまミヽヾ"   WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "あいうぇおざじたちなにぱぴまみゝゞ",
    "types": [
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "null"
    ],
    "checks": []
  }
]
